src/model/pyproject.toml

```diff
@@ -24,4 +24,5 @@ dependencies = [
     "datasets>=2.12.0",
     "zstandard>=0.21.0",
     "protobuf==3.20.0",
+    "lightning==2.0.2",
```

Comment:
You should not fix the dependency version. You should set flexible dependency for example `lightning>=2.0.2`

Reply:
done